src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -199,7 +188,9 @@ protected override void OnInitializeConfig(ScriptHostConfiguration config)
             var hostId = hostConfig.HostId ?? "default";
             Func<string, FunctionDescriptor> funcLookup = (name) => this.Instance.GetFunctionOrNull(name);
             var loggingConnectionString = config.HostConfig.DashboardConnectionString;
-            var instanceLogger = new FunctionInstanceLogger(funcLookup, _metricsLogger, hostId, loggingConnectionString, config.TraceWriter);
+
+            // TODO: This is asking for a LoggerFactory before the LoggerFactory is ready. Pass a Null instance for now.
```

Comment:
When will it get a non-null logger then? The bug I just fixed also involved the injection of null loggers and it was a subtle issue to track down what was going on.

Reply:
This is definitely a bug (I filed it yesterday: https://github.com/Azure/azure-webjobs-sdk-script/issues/2223). The only logging this thing does is on an exception, which is usually storage throttling. 

It's going to be a refactor to make this thing get initialized with the rest of our dependencies and I was hoping to punt it and not get bogged down there when the logging is pretty insignificant, especially in a preview build.

Reply:
... although I can just make sure the standby logger only uses a SystemLogger (without file logging) if it's an issue. It's just one more logging pipeline that you need to reason through when trying to figure out where the logs go.

Reply:
Ok, as long as you're tracking the bug. Perhaps put the link to the bug in here